content/en/docs/How To/create_upload.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Create a custom upload component"
+date: 2020-11-11
+weight: 750
+---
+The last (optional) component of a Baker pipeline is the Upload, whose job is to, precisely,
+upload somewhere what the output component produces.
```

Comment:
I think it should be clear from the start that the upload **must** deal with absolute paths to the local filesystem. There's no way around that, since that's how outputs that produces files work, and that's how the S3 upload works.
If I was to write an output which doesn't write absolute paths to the upload channel, then we'd have a problem since this output couldn't be placed before a S3 upload, which would defeat the modular purpose of baker component.